libvmaf/src/feature/integer_vif.c

```diff
@@ -37,758 +40,800 @@
 #endif
 #endif
 
-typedef struct VifState {
-    VifBuffer buf;
-    uint16_t log2_table[65537];
-    bool debug;
-    double vif_enhn_gain_limit;
-    void (*filter1d_8)(VifBuffer buf, unsigned w, unsigned h);
-    void (*filter1d_16)(VifBuffer buf, unsigned w, unsigned h, int scale,
-                        int bpc);
-    void (*filter1d_rd_8)(VifBuffer buf, unsigned w, unsigned h);
-    void (*filter1d_rd_16)(VifBuffer buf, unsigned w, unsigned h, int scale,
-                           int bpc);
-} VifState;
-
 static const VmafOption options[] = {
-    {
-        .name = "debug",
-        .help = "debug mode: enable additional output",
-        .offset = offsetof(VifState, debug),
-        .type = VMAF_OPT_TYPE_BOOL,
-        .default_val.b = false,
-    },
-    {
-        .name = "vif_enhn_gain_limit",
-        .help = "enhancement gain imposed on vif, must be >= 1.0, "
-                "where 1.0 means the gain is completely disabled",
-        .offset = offsetof(VifState, vif_enhn_gain_limit),
-        .type = VMAF_OPT_TYPE_DOUBLE,
-        .default_val.d = DEFAULT_VIF_ENHN_GAIN_LIMIT,
-        .min = 1.0,
-        .max = DEFAULT_VIF_ENHN_GAIN_LIMIT,
-    },
-    { 0 }
```

Comment:
This is an example of a whitespace only change.  It shows up in the diff, but nothing is changed. Avoid these, they're all over the place.